docs/REST API/rest-api.md

```diff
@@ -9288,7 +9298,7 @@ Retrieve details about a vulnerability and its affected software and OS versions
       "id": 2363,
       "name": "Docker Desktop",
       "version": "4.9.1",
-      "source": "programs",
+      "source": "programs", #ios,ipados
```

Comment:
@noahtalerman ah, one more – do we want these to be `_apps` as well?

Reply:
Yes! Good catch. I updates this